packages/playwright-core/src/utils/hostPlatform.ts

```diff
@@ -68,13 +68,13 @@ export const hostPlatform = ((): HostPlatform => {
         return ('ubuntu20.04' + archSuffix) as HostPlatform;
       return ('ubuntu22.04' + archSuffix) as HostPlatform;
     }
-    if (distroInfo?.id === 'debian' && distroInfo?.version === '11')
+    if ((distroInfo?.id === 'debian' || distroInfo?.id === 'raspbian') && distroInfo?.version === '11')
```

Comment:
I don't think we are comfortable making this change. We do not test on raspbian and so cannot say that it works. We should treat raspbian the same as any other distribution that is not officially supported.

Reply:
For what it's worth, I was imitating the pop/neon/tuxedo above; I assume they have equally valid reasons for altering the ID line in os-release. I'd assume that the firefox build for debian would be preferable to the ubuntu one but have not tested it.